tests/specs/expressions/ParenExpr/ParenExpr_All.txt

```diff
@@ -26,6 +26,18 @@ const test = (
 [expect]
 (x.test as unknown) = 6;
 
+== should not keep paren expr on right hand side of assignment (only) ==
+const a = (1);
+const b = ((((1))));
+const c = (1 + 1) + 1;
```

Comment:
Prettier actually collapses this so it must be doing some commutativity checking - something that could be done as a future improvement!

Reply:
I'm not sure about removing parens in math operations because some people like to add them to improve readability and make something obvious (I do sometimes). Maybe it could be done in a config setting.

Reply:
Yep, I can see that too 👍 especially if it's not as simple as consts like `1 + 1` but `(someVar + someOtherVar) + somethingCompletelyDifferent`